src/components/ContentHighlights/HighlightStepper/ContentConfirmContentCard.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { Delete } from '@edx/paragon/icons';
+import { IconButton, Icon } from '@edx/paragon';
+import { connect } from 'react-redux';
+import ContentHighlightCardItem from '../ContentHighlightCardItem';
+import { useContentHighlightsContext } from '../data/hooks';
+import { generateAboutPageUrl } from '../data/constants';
+
+const ContentConfirmContentCard = ({ enterpriseSlug, original }) => {
+  const { deleteSelectedRowId } = useContentHighlightsContext();
+  const [deleteKey, setDeleteKey] = useState(null);
+  const [isDeleted, setIsDeleted] = useState(false);
+  const {
+    title,
+    contentType,
+    partners,
+    cardImageUrl,
+    originalImageUrl,
+    firstEnrollablePaidSeatPrice,
+    aggregationKey,
+    key,
+  } = original;
+  // eslint-disable-next-line no-shadow
```

Comment:
[curious] why this disabling this ESLint rule necessary?

Reply:
leftover remnants of poorly written code, currently has no effect removing it

Reply:
Thought I did, now removed  👍🏼